module/API/Controller.cpp

```diff
@@ -1,12 +1,17 @@
 /**
  * @file Controller.cpp
  * @brief モーター制御に用いる関数をまとめたラッパークラス
- * @author takahashitom
+ * @author takahashitom CHIHAYATAKU
  */
 #include "Controller.h"
 
 Controller::Controller() : rightWheel(PORT_B), leftWheel(PORT_C), armMotor(PORT_A) {}
 
+// PWMの初期化
+double Controller::pwmOfRightWheel = 0.0;
+double Controller::pwmOfLeftWheel = 0.0;
+double Controller::pwmOfArm = 0.0;
+
 // モータに設定するPWM値の制限
 int Controller::limitPwmValue(const int value)
```

Comment:
```suggestion
int Controller::limitPwmValue(const doble value)
```
PWM値関連を全部 double にしちゃっていいかも。